gem2rpm/smart_proxy_plugin.spec.erb

```diff
@@ -26,27 +26,27 @@ Source0: <%= download_path %>%{gem_name}-%{version}.gem
 Requires: foreman-proxy >= %{foreman_proxy_min_version}
 Requires: ruby(release)
 <% for req in spec.required_ruby_version -%>
-Requires: ruby <%= req %>
+Requires: %{?scl_prefix}ruby <%= req %>
 <% end -%>
 <% for req in spec.required_rubygems_version -%>
-Requires: ruby(rubygems) <%= req %>
+Requires: %{?scl_prefix}ruby(rubygems) <%= req %>
 <% end -%>
 <% for d in spec.runtime_dependencies -%>
 <% for req in d.requirement -%>
-Requires: rubygem(<%= d.name %>) <%= req  %>
+Requires: %{?scl_prefix_ruby}rubygem(<%= d.name %>) <%= req  %>
```

Comment:
It could come from `_ror` as well. Could you cope the lookup that we have in the other SCL templates?